### PR TITLE
Allow customization of application verbiage.

### PR DESCRIPTION
**Pull request by samussiah**
i.e. application name, description, and image

### CODE REVIEW

File: R/safetyGraphicsApp.R [hunk 1]
```diff
@@ -15,39 +15,78 @@
...
+  mapping = NULL,
+  autoMapping = TRUE,
+  filterDomain = "dm",
   chartSettingsPaths = NULL,
-  runNow = TRUE
+  runNow = TRUE,
+  appName = 'safetyGraphics',
+  hexPath = system.file("resources/safetyGraphicsHex.png", package = "safetyGraphics"),
+  homeTabPath = system.file('resources/safetyGraphicsHomeTab.html', package = 'safetyGraphics'),
+  launchBrowser = FALSE
```
Comment (jwildfire) [inline R/safetyGraphicsApp.R]: Need to add new params in header

File: R/safetyGraphicsApp.R [hunk 2]
```diff
@@ -15,39 +15,78 @@
...
+
+  if (runNow) {
+    if (launchBrowser == TRUE) {
+      runApp(app, launch.browser = TRUE)
+    } else {
+      runApp(app)
+    }
+  }
+
+  app
```
Comment (jwildfire) [inline R/safetyGraphicsApp.R:85]: Need to try a deploy to shinyapps.io with this update. IIRC the previous code was added to make that work cleanly.

File: R/safetyGraphicsApp.R (hunk 1 above)
Comment (jwildfire) [reply R/safetyGraphicsApp.R]: Done.

File: R/safetyGraphicsApp.R [hunk 3]
```diff
@@ -15,39 +15,78 @@
...
-  config <- app_startup(domainData, meta, charts, mapping, autoMapping, filterDomain, chartSettingsPaths)
+
+  if (!is.character(appName))
+    appName <- 'safetyGraphics'
+
+  if (!is.character(hexPath) || !file.exists(hexPath))
+    hexPath <- system.file("resources/safetyGraphicsHex.png", package = "safetyGraphics")
+
+  if (!is.character(homeTabPath) || !file.exists(homeTabPath))
+    homeTabPath <- system.file('resources/safetyGraphicsHomeTab.html', package = 'safetyGraphics')
```
Comment (jwildfire) [inline R/safetyGraphicsApp.R]: Moved this logic to `app_startup()`

**Review by jwildfire — APPROVED** [review]

File: R/safetyGraphicsApp.R (hunk 2 above)
Comment (samussiah) [reply R/safetyGraphicsApp.R:85]: Captured in #708

**Pull request merged**